chcemvediet/apps/inforequests/models/action.py

```diff
@@ -345,7 +345,7 @@ def is_by_email(self):
 
     @cached_property
     def has_obligee_deadline(self):
-        return self.deadline and self.deadline.is_obligee_deadline
+        return bool(self.deadline) and self.deadline.is_obligee_deadline
```

Comment:
@martinmacko47 Pozmenil som funkcionality `has_*_deadline`. Ako z nazvu funkcie vyplyva na vystupe ocakavam True/False. Je to tak aj spoznamkovane v dokumentacii `inforequest.md`. Vsade v kode kde sa pouzival tieto metody, to ich cinnost nezmeni.

Reply:
@viliambalaz Ok, ale nebolo by citatelnejsie skor `self.deadline is not None`? Ci aj v inych pripadoch potrebujes aby to bolo false?

Reply:
:ok: